internal/clients/diffreviewer/diffutils/diff_filter.go

```diff
@@ -0,0 +1,46 @@
+package diffutils
+
+import (
+	"strings"
+
+	"github.com/nightfallai/nightfall_code_scanner/internal/clients/diffreviewer"
+)
+
+func FilterFileDiffs(fileDiffs []*diffreviewer.FileDiff) []*diffreviewer.FileDiff {
```

Comment:
all of these funcs were copy/pasta from github package right? Nothing net new?

Reply:
yup I just pulled them out into their own package